yarn.lock

```diff
@@ -114,19 +114,19 @@
     cloudform "^4.2.0"
     constructs "^3.3.125"
     fs-extra "^8.1.0"
-    graphql "^14.5.8"
-    graphql-auth-transformer "7.2.47"
-    graphql-connection-transformer "5.2.46"
-    graphql-dynamodb-transformer "7.2.46"
-    graphql-elasticsearch-transformer "5.2.47"
-    graphql-function-transformer "3.3.37"
-    graphql-http-transformer "5.2.46"
-    graphql-key-transformer "3.2.46"
-    graphql-predictions-transformer "3.2.46"
-    graphql-relational-schema-transformer "2.21.13"
-    graphql-transformer-common "4.24.3"
-    graphql-transformer-core "7.6.9"
-    graphql-versioned-transformer "5.2.46"
+    graphql "^15.5.0"
```

Comment:
did graphql update to 15 go with `10.7.1` cli? 
if not then we should bundle this hotfix with 10.8 or downgrade grapqhl before merging.

Reply:
GraphQL 15 change is not included with `10.7.1`. Another option would be to include the GraphQL 15 upgrade in this hotfix. 

https://github.com/aws-amplify/amplify-cli/pull/1175
https://github.com/aws-amplify/amplify-cli/pull/11893